src/super_gradients/training/transforms/keypoints/keypoints_mosaic.py

```diff
@@ -0,0 +1,132 @@
+import random
+import cv2
+import numpy as np
+
+from typing import Optional, List
+
+from super_gradients.common.registry import register_transform
+from super_gradients.training.samples import PoseEstimationSample
+from .abstract_keypoints_transform import AbstractKeypointTransform
+
+
+@register_transform()
+class KeypointsMosaic(AbstractKeypointTransform):
+    """
+    Mix two samples together.
+
+    :attr prob:            Probability to apply the transform.
```

Comment:
Docs are wrong here.

Reply:
Fixed in c7971002071494f217d38258a9466266aa1ba38f